pkg/sosreport/index.jsx

```diff
@@ -265,7 +265,7 @@ const SOSDialog = () => {
         }
 
         if (verbose) {
-            args.push("-vvv");
+            args.push("-v");
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-17571-20220718-095624-44b4461f-fedora-36-devel//Coverage/lcov/github-pr.diff.gcov.html)